AppConnectSwift/FormListViewController.swift

```diff
@@ -123,22 +128,30 @@ class FormListViewController: UITableViewController {
         // Start a view controller to fill out the form. If the form is from the SDK
         // sample CRF, we open FORM1 as a one-page form and FORM2 as a multi-page
         // form to demonstrate how to handle both cases.
-        let form = objects[indexPath.row] as! MDForm
-        let sequeIdentifier = ["FORM1" : "ShowOnePageForm", "FORM2" : "ShowMultiPageForm"][form.formOID]
-        performSegueWithIdentifier(sequeIdentifier!, sender: self)
+        if(indexPath.row == 2){
+            captureImage(self)
+        }
```

Comment:
Converted but need to test this once Rave study is up for sandbox or validation.